### ISSUE TITLE
Can labels be put into iframes for better accessibility?

### ISSUE THREAD

**Issue by SteveDoesStuff** [recurly/recurly-js#283]
When using a screen-reader (ie. VoiceOver on Mac) the input fields within the iframes have no labels associated to them that can be read, so they are just called "required. edit text" - ideally, it should read something like "Credit card number. required. edit text" so the user knows the field's purpose. Would it be possible to either:
1. Define a label element in recurly.configure() that would be placed within the associated iframe above/below the input element
2. Define an aria-label attribute value in recurly.configure() that would be added to the associated input element

I've seen #279 addresses the UX side of the label/input pairing, but the accessibility side is still an issue

**Comment by chrissrogers** [comment]
Great suggestions on accessibility considerations. We currently set the `title` attribute to match the placeholder value given to a field, which screen readers should dictate on the field. However, if a placeholder value is not desired for the visual style of the field, then no title is set.

What do you think about setting the `title` and `aria-title` from a configurable title value separate from the placeholder content value?

**Comment by SteveDoesStuff** [comment]
Setting the title sounds like a good solution. Oddly VoiceOver isn't reading the aria-title on an input - it reads aria-label just fine though.

In [this article by WebAIM](http://webaim.org/techniques/forms/advanced#arialabel) they recommend using either the title **or** aria-label attribute to avoid the same text being read twice, so I think setting just the title value would be best so there's no potential for duplication by introducing the aria-label.

**Comment by chrissrogers** [comment]
Perfect. `title` it shall be then. Now, if you're looking for a solution to use immediately, you may consider a workaround in which you set the placeholder content (thereby setting the title value), but if stylistically you wish to not have a placeholder, then you may color the placeholder identical to the background of the input. It's certainly not graceful, but will let you achieve accessibility immediately without having to wait for this to be published.

**Comment by SteveDoesStuff** [comment]
Brilliant! Thanks Chris, that's definitely a workable solution for now. Looking forward to this and your solution to giving focus to inputs by clicking labels to be implemented

**Comment by chrisspadanuta** [comment]
I haven't been able to find "title" in the documentation, but if I set title, I do indeed see it setting aria-label and title on the input field inside the iframe.  Is this officially a feature, could the docs be updated or did I just miss it in the docs?
